lib/components/molecules/ArticleThumbnail/ArticleThumbnail.js

```diff
@@ -12,7 +12,12 @@ import type { Props } from './types';
 import Image from '../../atoms/Image/Image';
 import Anchor from '../../atoms/Anchor';
 
-const getAnchoredChild = (anchorProps, children) => <Anchor {...anchorProps}>{children}</Anchor>;
+const i = 0;
+const getAnchoredChild = (anchorProps, children) => (
```

Comment:
Why you have these changes part of your PR?

Reply:
all changes have been roll backed